### PR TITLE
Upgraded Ember to 2.6.0

### PR DESCRIPTION
**Pull request by rkrishnan8594**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015
+Copyright (c) 2016
```
Comment (rsocci) [inline LICENSE.md]: As I learned from @michaeldupuisjr , this should still be 2015 to make sure everything for that year is also copyrighted

File: package.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "ember-cli-test-assertions",
-  "version": "0.0.1",
+  "version": "0.0.0",
```
Comment (rsocci) [inline package.json]: Oopsie, version should not change

File: LICENSE.md (hunk 1 above)
Comment (rkrishnan8594) [inline LICENSE.md]: Interesting, did not know that!

**Comment by rsocci** [comment]
Just a couple of comments, LGTM otherwise!

**Comment by michaeldupuisjr** [comment]
LGTM

**Pull request merged**